demo/wdio.conf.js

```diff
@@ -22,13 +12,14 @@ const config = {
   // Custom settings
   // ===============
   logLevel: 'info', // trace | debug | info | warn | error | silent
-  outputDir: './_results_',
+  outputDir: path.join(process.cwd(), `./_results_`),
```

Comment:
Just in case/for reference:
when using `__dirname` instead of `process.cwd()` in ES scope you'll get something like 
```
ReferenceError: __dirname is not defined in ES module scope
```
Alternative to `process.cwd()` might be:

```js
import { fileURLToPath } from 'url'
import { dirname } from 'path'
const __filename = fileURLToPath(import.meta.url)
const __dirname = dirname(__filename)
```
⬆️ looks terrible imho